app/src/main/res/layout/account_history_cell.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="5dp"
+    app:cardCornerRadius="25dp"
+    app:cardElevation="0dp">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/relativeLayoutAccountHistory"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="5dp"
+        android:background="@color/orange_200"
+        android:padding="12dp">
+
+        <TextView
+            android:id="@+id/accountHistoryAccount_Value"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:gravity="center"
+            android:text="Account"
+            android:textSize="15dp"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:hint="account" />
+
+
+        <TextView
+            android:id="@+id/accountHistoryDate_Value"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:gravity="center"
+            android:textSize="15dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:hint="Date" />
+
+        <TextView
+            android:id="@+id/accountHistoryBalance_Title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="5dp"
+            android:gravity="left"
+            android:text="Balance:  "
+            android:textSize="15dp"
+            app:layout_constraintEnd_toStartOf="@+id/accountHistoryBalance_Value"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/accountHistoryAccount_Value" />
+
+        <TextView
```

Comment:
![image](https://user-images.githubusercontent.com/22043527/114641830-cba1a180-9ca0-11eb-8c75-ac0cbdcdff20.png)
The alignment on the 'Balance' and '% Change' columns should be at the endpoint of these two texts. The two should be right aligned.

Reply:
Also please right align the balance textviews as well

Reply:
please check if what I updated is what you are looking for